randovania/games/samus_returns/exporter/hint_namer.py

```diff
@@ -10,11 +10,22 @@
 
 if TYPE_CHECKING:
     from randovania.exporter.hints.pickup_hint import PickupHint
+    from randovania.game_description.db.pickup_node import PickupNode
+    from randovania.game_description.db.region_list import RegionList
     from randovania.game_description.game_patches import GamePatches
     from randovania.games.game import RandovaniaGame
     from randovania.interface_common.players_configuration import PlayersConfiguration
 
 
+def _area_name(region_list: RegionList, pickup_node: PickupNode, hide_region: bool) -> str:
+    area = region_list.nodes_to_area(pickup_node)
+    return region_list.area_name(area)
+
+
+def colorize_text(text: str, with_color: bool) -> str:
+    return text
```

Comment:
This looks useless if it only returns the first parameter without doing anything.
Could also keep it for future if we find out how to colorize the text.

Reply:
I believe I left that because the function was being called further down. Could probably simplify that or just leave it.